src/core_plugins/kibana/public/dashboard/dashboard.js

```diff
@@ -25,6 +26,8 @@ import { notify } from 'ui/notify';
 import './panel/get_object_loaders_for_dashboard';
 import { documentationLinks } from 'ui/documentation_links/documentation_links';
 import { showCloneModal } from './top_nav/show_clone_modal';
+import { kueryASTUtils, toKueryExpression, fromKueryExpression, filterToKueryAST } from 'ui/kuery';
+import { migrateLegacyQuery } from '../../../../ui/public/utils/migrateLegacyQuery.js';
```

Comment:
It can be, but these webpack aliases give me indigestion. I suppose I could switch it for the sake of consistency with this existing file.